_posts/2022-11-28-trino-summit-2022-apple-recap.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Trino at Apple"
+author: "Vinitha Gankidi, Yathi Peddyshetty, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/apple.jpg
+---
+
+This post continues [a larger series of posts]
+({\% post_url 2022-11-21-trino-summit-2022-recap %}) on the Trino Summit
+2022 sessions. Following the [Keynote: State of Trino session]
+({\% post_url 2022-11-22-trino-summit-2022-keynote-recap %}), engineers from
```

Comment:
intentionally added a slash to avoid jekyll from freaking out. Will remove these once both blogs are published